packages/react-components/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -143,6 +140,22 @@ const useInputStyles = makeStyles({
       },
     },
   },
+
+  before: {
+    right: 0,
+  },
+  after: {
+    left: 0,
+  },
+
+  // Calculate the width of the hidden input by taking into account the size of the indicator + the padding around it.
+  // This is done so that clicking on that "empty space" still toggles the checkbox.
+  medium: {
+    width: `calc(${indicatorSizeMedium} + ${tokens.spacingHorizontalS})`,
+  },
+  large: {
+    width: `calc(${indicatorSizeLarge} + ${tokens.spacingHorizontalS})`,
+  },
```

Comment:
I think you can get rid of `spacingHorizontalS` now that the label is using padding instead of margin for the gap, right?
```suggestion
  medium: {
    width: indicatorSizeMedium,
  },
  large: {
    width: indicatorSizeLarge,
  },
```

Reply:
No, that spacing is to account for the space on the left of the `indicator` (right if `labelPosition` is `before`). You can see that another `tokens.spacingHorizontalM` has been removed already that referred to the gap between the `indicator` and the `label`.